### DISCUSSION TITLE
Problems with rendering large amounts of text

### DISCUSSION THREAD

**Discussion by AllesOderNicht** [remarkjs/remark#1267] [Q&A]
How should I optimize when I have a lot of text to render

**Comment by ChristianMurphy** [comment]
Welcome @AllesOderNicht! 👋 

Optimization depends on what you need to do.
There are many ways to use remark: as a lint tool, as a transformation tool, as a rendering tool, etc.
It can also be used through many libraries, on many runtime platforms, and combined with different frameworks.
There is no one size fit all "optimize" approach, you need to be more specific on your needs.

You can find more on how to help us help you with your question at https://github.com/remarkjs/.github/blob/main/support.md
